src/less/variables/ds6/icon-variables.less

```diff
@@ -398,8 +398,8 @@
 @icon-arrow-left-small-disabled-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLWFycm93LWxlZnQtc21hbGwtZGlzYWJsZWQtYmFzZTY0IiB2aWV3Qm94PSIwIDAgMTQgMTQiPjxwYXRoIGZpbGw9IiNjN2M3YzciIGQ9Ik0xMyA2LjAwNEgzLjQxbDQuMy00LjI5QTEuMDA0IDEuMDA0IDAgMCAwIDYuMjkuMjk0bC02IDZhMSAxIDAgMCAwIDAgMS40MWw2IDZhMSAxIDAgMCAwIDEuNDEtMS40MWwtNC4yOS00LjI5SDEzYTEgMSAwIDAgMCAwLTJ6Ii8+PC9zdmc+";
 @icon-arrow-right-disabled-base64: "PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGlkPSJpY29uLWFycm93LXJpZ2h0LWRpc2FibGVkLWJhc2U2NCIgdmlld0JveD0iMCAwIDIwIDE4Ij48cGF0aCBmaWxsPSIjYzdjN2M3IiBkPSJNMTkuNzEgOC4yOTRsLTgtOGExLjAwNCAxLjAwNCAwIDAgMC0xLjQyIDEuNDJsNi4zIDYuMjlIMWExIDEgMCAwIDAgMCAyaDE1LjU5bC02LjMgNi4yOWExLjAwMSAxLjAwMSAwIDAgMCAwIDEuNDIgMS4wMDEgMS4wMDEgMCAwIDAgMS40MiAwbDgtOGExLjAwMSAxLjAwMSAwIDAgMCAwLTEuNDJ6Ii8+PC9zdmc+";
```

Comment:
If you can double check and see if the disabled background icons are proper. I'm can't remember if I changed them or not, but I have the feeling that I did not.

Reply:
Sure, thanks for reminding me. I only updated the active one for the color.